pkg/sbom/v1/sbom_spdx_storage_format.go

```diff
@@ -86,6 +95,11 @@ func (sbom *SBOMData) saveSBOM(spdxData *spdxv1beta1.SBOMSPDXv2p3) error {
 	return nil
 }
 
+func parsedFilesBySourceInfo(packageSourceInfo string) []string {
+	fileListInString := utils.After(packageSourceInfo, ": ")
+	return strings.Split(fileListInString, ", ")
+}
+
 func (sbom *SBOMData) StoreSBOM(sbomData any) error {
```

Comment:
We are making here such a big change, how is it that no unit tests are effected?
I saw you added a test for the `parsedFilesBySourceInfo` function, but I will expect tests for `StoreSBOM` as well.